apikana-maven-plugin/src/main/java/org/swisspush/apikana/AbstractApikanaMojo.java

```diff
@@ -158,10 +158,18 @@ protected boolean isPom() {
     protected void generatePackageJson(String version) throws IOException {
         updateJson(working("package.json"), pack -> {
             pack.put("name", mavenProject.getArtifactId());
+            pack.put("version", mavenProject.getVersion());
```

Comment:
I strongly suggest to NOT doing this. See https://github.com/swisspush/apikana-java/pull/8

Reply:
Projects not using semver can now override the version using apikana.apiversion